sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/CHANGELOG.md

```diff
@@ -1,14 +1,31 @@
 # Release History
 
-## 1.7.0-beta.2 (Unreleased)
+## 1.7.0-beta.2 (2022-02-14)
 
 ### Features Added
-
-### Breaking Changes
+* `SignalRConnectionAttribute` can be used to customize the connection name of strongly typed serverless hub too. Usage:
+    ```cs
+    [SignalRConnection("SignalRConnection")]
+    public class CustomConnectionHub : ServerlessHub<IChatClient>
+    {
+    }
+    ```
+    ```json
+    {
+    //local.settings.json
```

Comment:
```suggestion
```

Comments are not valid JSON elements.  _(see [JSON standard](https://www.json.org))_